content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-reconfigure.md

```diff
@@ -204,16 +204,8 @@ kubectl edit service -n kube-system kube-dns
 
 Once the CoreDNS changes are applied you can delete the CoreDNS Pods:
 
-Obtain the Pod names:
-
-```shell
-kubectl get po -n kube-system | grep coredns
-```
-
-Delete a Pod with:
-
 ```shell
-kubectl delete po -n kube-system <pod-name>
+kubectl delete po -n kube-system -l k8s-app=kube-dns
```

Comment:
Thinking about this more: this deletes _all_ Pods, not one Pod.

How about [`kubectl rollout restart`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_rollout/kubectl_rollout_restart/)?

Reply:
edc4dc7e493798cf6c1db40807be5ca90e91c9b4